polaris.shopify.com/content/patterns/app-settings-layout.ts

```diff
@@ -4,7 +4,7 @@ const pattern: SingleVariantPattern = {
   title: 'App settings layout',
   description:
     'Lets merchants easily scan many groups of settings and find the ones they want to change.',
-  howItHelps: `![A labeled diagram of an app settings page layout. The left column is labeled "1" it contains glanceable labels and descriptions. The right column is labeled "2" and contains a list of cards.](/images/patterns/app-settings-cover-image.png)
+  howItHelps: `![App settings page with two columns.](/images/patterns/app-settings-cover-image.png)
```

Comment:
I just realized seeing it here in the code that we should align on **not** using an ending period on these short alt-text descriptions. In the source doc, we sometimes had them and sometimes didn't. But given that they are all short sentences, I think it's best to remove them.